stock_picking_auto_create_lot/tests/test_stock_picking_auto_create_lot.py

```diff
@@ -144,3 +144,17 @@ def test_multi_auto_create_lot(self):
             [("product_id", "=", self.product_serial.id)]
         )
         self.assertEqual(len(lot), 6)
+
+    def test_auto_create_lot_with_quantity_done(self):
+        self.picking.action_assign()
+        move = self.picking.move_ids.filtered(
+            lambda m: m.product_id == self.product_serial
+        )
```

Comment:
```suggestion
        )
        self.assertEqual(move.product_uom_qty, 3.0)
```
To make the point of the test clearer.